models/nexusmutual/ethereum/nexusmutual_ethereum_product_information.sql

```diff
@@ -0,0 +1,178 @@
+{{ config(
+    alias ='product_information',
+    materialized='view',
+    unique_key = ['product_contract_address', 'product_name', 'product_type', 'syndicate', 'date_added'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "nexusmutual",
+                                \'["guyhowlett"]\') }}'
+    )
+}}
+
+SELECT lower('0xF5DCe57282A584D2746FaF1593d3121Fcac444dC') as product_contract_address, 'Compound Sai' as product_name,  'protocol' as product_type, '2020-01-01' as date_added, 'v1' as syndicate UNION ALL
+SELECT lower('0x8B3d70d628Ebd30D4A2ea82DB95bA2e906c71633') as product_contract_address, 'bZx v1' as product_name,  'protocol' as product_type, '2020-01-01' as date_added, 'v1' as syndicate UNION ALL
+SELECT lower('0x5504a1d88005236147EC86C62cfb53043bD1276a') as product_contract_address, 'Unknown' as product_name,  'protocol' as product_type, '2020-01-01' as date_added, 'v1' as syndicate UNION ALL
+SELECT lower('0x4Ddc2D193948926D02f9B1fE9e1daa0718270ED5') as product_contract_address, 'Compound ETH' as product_name,  'protocol' as product_type, '2020-01-01' as date_added, 'v1' as syndicate UNION ALL
+SELECT lower('0x080bf510FCbF18b91105470639e9561022937712') as product_contract_address, '0x v2.1' as product_name,  'protocol' as product_type, '2020-01-01' as date_added, 'v1' as syndicate UNION ALL
+SELECT lower('0x16de59092dAE5CcF4A1E6439D611fd0653f0Bd01') as product_contract_address, 'iearn yDAI v1' as product_name,  'protocol' as product_type, '2020-01-01' as date_added, 'v1' as syndicate UNION ALL
+SELECT lower('0xAF350211414C5DC176421Ea05423F0cC494261fB') as product_contract_address, 'Saturn DAO Token' as product_name,  'protocol' as product_type, '2020-01-01' as date_added, 'v1' as syndicate UNION ALL
```

Comment:
Hi @guyhow, since there is no `select xxx from xxx`, we can rewrite the hardcoded values in `SELECT xxx FROM (
VALUES ...` format.

Reply:
done